src/Init/Data/Int/LemmasAux.lean

```diff
@@ -140,4 +140,34 @@ theorem bmod_bmod_of_dvd {a : Int} {n m : Nat} (hnm : n ∣ m) :
   obtain ⟨k, rfl⟩ := hnm
   simp [Int.mul_assoc]
 
+theorem toInt_sub_toInt_lt_twoPow_iff {x y : Int} {k : Nat}
```

Comment:
```suggestion
private theorem toInt_sub_toInt_lt_twoPow_iff {x y : Int} {k : Nat}
```

Reply:
this is a bit too generic :D I was under the impression that we would generalize it to `(x y : BitVec w)`, and write theorems about `x.toInt` and `y.toInt`. 

If we write this theorem with this generality, it feels a bit too "special case" for our purpose: I can't think of anyone reasoning about integers who needs this theorem. So, I'd suggest moving the theorem to `src/Init/Data/BitVec/Bitblast.lean` as a `private theorem.

Reply:
I see your point! I'd rather write te theorem about `x.toInt` and `y.toInt`, since it seems more usefuil.